src/pca_gophish.yml

```diff
@@ -120,7 +120,31 @@
                 enabled: true
           when:
             - ansible_service_mgr == "systemd"
-    - name: Create GoPhish desktop shortcut for VNC user
+    - name: Create standard directories used by PCA operators
+      block:
+        - name: Create data directories
+          ansible.builtin.file:
+            group: "{{ vnc_username }}"
+            mode: 0755
+            owner: "{{ vnc_username }}"
+            path: /home/{{ vnc_username }}/PCA/Level-{{ item }}/Data
+            state: directory
+            # range is exclusive of end point
+          loop: "{{ range(1, 7)|list }}"
```

Comment:
Magic number. 🧙

Reply:
Feast your 👀  on this: https://github.com/cisagov/pca-gophish-composition-packer/pull/55/commits/2f413c2ea92d847110bd286acdeda13fda31caf1